### PR TITLE
PP-4862: Modify provider tests

### PR DESCRIPTION
**Pull request by oswaldquek**
Changes to get https://github.com/alphagov/pay-publicapi/pull/387/ passing.

### CODE REVIEW

File: src/test/java/uk/gov/pay/connector/it/resources/SearchChargesITest.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    @Before
+    public void setupGatewayAccount() {
+        databaseTestHelper = testContext.getDatabaseTestHelper();
+        accountId = String.valueOf(nextLong());
+        databaseTestHelper.addGatewayAccount(accountId, "sandbox");
+        connectorRestApiClient = new RestAssuredClient(testContext.getPort(), accountId);
+    }
+
+    @Test
+    public void searchChargesReturnsExpectedValues() {
```
Comment (kbottla) [inline src/test/java/uk/gov/pay/connector/it/resources/SearchChargesITest.java]: Could the method be named to something like `searchChargesReturnsExpectedGatewayTransactionId` to make the test name clear.

**Review by kbottla — COMMENTED** [review]
LGTM, just added a minor comment about test method naming

File: src/test/java/uk/gov/pay/connector/it/resources/SearchChargesITest.java (hunk 1 above)
Comment (oswaldquek) [reply src/test/java/uk/gov/pay/connector/it/resources/SearchChargesITest.java]: fixed! thanks for reviewing

**Review by kbottla — APPROVED** [review]

File: src/test/java/uk/gov/pay/connector/it/resources/SearchChargesITest.java (hunk 1 above)
Comment (kbottla) [reply src/test/java/uk/gov/pay/connector/it/resources/SearchChargesITest.java]: 👍 Thanks!

**Pull request merged**